packages/common/src/services/wallet-client/WalletClient.ts

```diff
@@ -59,13 +59,18 @@ export class WalletClient {
   }
 
   /** Get user's current SOL Audio balance. Returns null on failure. */
-  async getCurrentWAudioBalance(ethAddress?: string): Promise<BNWei | null> {
```

Comment:
I'm honestly not sure why we need WalletClient if it's just a passthrough for the function in audiusBackendInstance.
Alternatively, I'm not sure why we shouldn't just do the work that's in ABE in this function?

Reply:
yeah you're right but i'm gonna leave the cleanup for later